sympy/integrals/risch.py

```diff
@@ -162,7 +173,8 @@ class DifferentialExtension(object):
     # only create one DifferentialExtension per integration).  Also, it's nice
     # to have a safeguard when debugging.
     __slots__ = ('f', 'x', 'T', 'D', 'fa', 'fd', 'Tfuncs', 'backsubs', 'E_K',
-        'E_args', 'L_K', 'L_args', 'cases', 'case', 't', 'd', 'newf', 'level',
+        'E_args', 'L_K', 'L_args', 'T_K', 'T_args', 'AT_K', 'AT_args', 'ALG_K',
```

Comment:
I think so. The argument list may have to be reconsidered when algebraic extensions are introduced, but now it could be a combination of the lists `E_args`, etc.